proto/cmp/services/accommodation/v3/property_types.proto

```diff
@@ -0,0 +1,198 @@
+
+  // Contact Info: phone, address, email, links
+  cmp.types.v2.ContactInfo contact_info = 8;
+
+  // Location coordinate
+  cmp.types.v2.Coordinates coordinates = 9;
+
+  // Status of the property
+  cmp.types.v1.ProductStatus status = 10;
+
+  // Airports
+  // Ex: ["PMI", "ZRH", "AYT"]
+  repeated string airports = 11;
+}
+
+enum CategoryRating {
+  CATEGORY_RATING_UNSPECIFIED = 0;
+  CATEGORY_RATING_0_5 = 1;
+  CATEGORY_RATING_1_0 = 2;
+  CATEGORY_RATING_1_5 = 3;
+  CATEGORY_RATING_2_0 = 4;
+  CATEGORY_RATING_2_5 = 5;
+  CATEGORY_RATING_3_0 = 6;
+  CATEGORY_RATING_3_5 = 7;
+  CATEGORY_RATING_4_0 = 8;
+  CATEGORY_RATING_4_5 = 9;
+  CATEGORY_RATING_5_0 = 10;
+  CATEGORY_RATING_5_5 = 11;
+  CATEGORY_RATING_6_0 = 12;
+  CATEGORY_RATING_6_5 = 13;
+  CATEGORY_RATING_7_0 = 14;
+  CATEGORY_RATING_7_5 = 15;
+  CATEGORY_RATING_8_0 = 16;
+  CATEGORY_RATING_8_5 = 17;
+  CATEGORY_RATING_9_0 = 18;
+  CATEGORY_RATING_9_5 = 19;
+  CATEGORY_RATING_10_0 = 20;
+}
+
+enum CategoryUnit {
+  CATEGORY_UNIT_UNSPECIFIED = 0;
+  CATEGORY_UNIT_STARS = 1;
+  CATEGORY_UNIT_PALMS = 2;
+}
+
+// This message type contains extended info about a property
+message PropertyExtendedInfo {
+  // Property
+  cmp.services.accommodation.v3.Property property = 1;
+
+  // Images
+  repeated cmp.types.v3.Image images = 2;
+
+  // Videos
+  repeated cmp.types.v3.Video videos = 3;
+
+  // Segmentation classification
+  repeated string classifications = 4;
+
+  // Property descriptions
+  repeated cmp.types.v1.LocalizedDescriptionSet localized_descriptions = 5;
+
+  // Room descriptions
+  repeated cmp.types.v1.LocalizedDescriptionSet localized_room_descriptions = 6;
+
+  // Payment type. Ex: "MERCHANT"
+  string payment_type = 7;
+
+  // Rooms
+  repeated cmp.services.accommodation.v3.Room rooms = 8;
+}
+
+message Room {
+  // Room code of the unit in case of hotel. Ex: "RMSDDB0000". For holiday homes
+  // often no room codes are used. This code must explicitly match the supplier_room_code
+  // in the ProductList and the ProductInfo messages, so that the static data like room
+  // amenities, descriptions, images etc. can be married to the dynamic price and
+  // availability data for display to the end user.
+  string supplier_code = 1;
+
+  // Room name. In case of hotel a standardized room name is often derrived from
+  // room code structures. Exmple: "superior seaview room".
+  // For holiday homes we expect names like "Master Bedroom", "Second Bedroom" or
+  // "Bathroom" when specific descriptions for each room are available.
+  string supplier_name = 2;
+
+  // Original room name as assigned by the hotel. In case of spefifically designated room
+  // names by chains and both the chain and the customer wants to adhere to them.
+  // Ex: "CEIBA CLUB", "CEIBA GOVERNORS SUITE", "GOVERNORS SUITE" or "ENCLAVE NATURE VIEW"
+  // For holiday homes specific room names if availbale can be given.
+  string original_name = 3;
```

Comment:
```suggestion
  // Room code of the unit in case of hotel. Ex: "RMSDDB0000". For holiday homes
  // often no room codes are used. This code must explicitly match the supplier_room_code
  // in the ProductList and the ProductInfo messages, so that the static data like room
  // amenities, descriptions, images etc. can be married to the dynamic price and
  // availability data for display to the end user.
  string supplier_code = 1;

  // Room name. In case of hotel a standardized room name is often derived from
  // room code structures. Example: "superior sea view room".
  // For holiday homes we expect names like "Master Bedroom", "Second Bedroom" or
  // "Bathroom" when specific descriptions for each room are available.
  string supplier_name = 2;

  // Original room name as assigned by the hotel. In case of specifically designated room
  // names by chains and both the chain and the customer wants to adhere to them.
  // Ex: "CEIBA CLUB", "CEIBA GOVERNORS SUITE", "GOVERNORS SUITE" or "ENCLAVE NATURE VIEW"
  // For holiday homes specific room names if available can be given.
  string original_name = 3;
```
Typos